vllm/v1/worker/tpu_model_runner.py

```diff
@@ -88,7 +88,9 @@ def __init__(
         self.max_model_len = model_config.max_model_len
         self.max_num_blocks_per_req = cdiv(self.max_model_len, self.block_size)
         self.max_num_tokens = scheduler_config.max_num_batched_tokens
-        self.max_num_reqs = scheduler_config.max_num_seqs
+        # InputBatch needs to work with sampling tensors greater than padding
+        # to avoid dynamic shapes. Also, avoid suboptimal alignment.
+        self.max_num_reqs = max(scheduler_config.max_num_seqs, MIN_NUM_SEQS)
```

Comment:
MIN_NUM_SEQS is 8, should this be rounding up to the nearest number divisible by MIN_NUM_SEQS?

Reply:
not necessarily see in `capture_model`, max_num_reqs still gets compiled anyway due to how padding with upper limit is implemented.

Reply:
btw I'm ok with forcing everything to be nicely divisible by `MIN_NUM_SEQS`; just I remember `max_num_seqs` used to be padded and then it was changed for reasons